### DISCUSSION TITLE
Welcome to Swiftide Discussions!

### DISCUSSION THREAD

**Discussion by timonv** [bosun-ai/swiftide#430] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shamb0** [comment]
Hello Swiftide Community,

Thanks @timnov for creating this project and fostering this community.

I discovered Swiftide while searching for RAG integration tools for my Obsidian PKM system. What drew me to the project is its RUST-based tech stack, support for local vector DBs and LLM models, and its well-architected design.

From my analysis, Swiftide's indexing and retrieval pipeline is optimized for memory efficiency and parallel processing capabilities, making it both scalable and performant within available computing resources.

I've started contributing to the project with my first PR (#392) currently under review.

I'm Rajeshkumar (Shamb0) from India - a tech evangelist and RUST enthusiast who contributes to RUST open source projects and early-stage startups.

Looking forward to building Swiftide together with the community.

**Reply by timonv** [reply]
Great to have you here!
